### PR TITLE
Add Vernier Force and Accel extension landing page

### PR DESCRIPTION
**Pull request by ericrosenbaum**
This adds a landing page and other support for the Vernier Force & Acceleration extension, including:
 
- Landing page at `/vernier`, with instructions, starter projects, troubleshooting info, etc.
- The extension chip to display under the project on the project page

Note that on staging, the starter project links will not work.

### CODE REVIEW

**Comment by ericrosenbaum** [comment]
Thanks @chrisgarrity! 

re: the inconsistent use of 'and' vs  '&', I thought about that and used the best compromise I could. The name of the extension appears in 6 distinct places, each with different constraints, and especially long name in this case makes it tricky. (Not in this PR, but I was not able to use "&" in the name in the blocks category menu and in the blocks menu, because of a known scratch-blocks bug, so I used "and" there). So I selected different versions of the name for each of those places, and reviewed them with the design team here and with folks at Vernier. Actually, what the heck- here's all of them:

Screen Shot 2019-04-10 at 12 25 58 PM

re: "Go Direct" vs. "GoDirect", I was explicitly told by folks at Vernier that we should use the version with no space in it, for branding reasons. This seems like it may be incorrect, but it also seems late to try to double check with them now, so I think we should leave as is and fix later if they request it.

**Comment by ericrosenbaum** [comment]
re: localization of "Force and/& Acceleration", we'll have to check with Vernier about what they would like- let's leave it unlocalized for now.

File: src/lib/extensions.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ const EXTENSION_INFO = {
     makeymakey: {
         name: 'Makey Makey',
         icon: 'extension-makeymakey.svg'
+    },
+    gdxfor: {
+        name: 'Force and Acceleration',
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/lib/extensions.js]: This matches the name of the extension used in the editor

File: src/routes.json [hunk 2]
```diff
@@ -295,6 +295,13 @@
         "view": "microbit/microbit",
         "title": "micro:bit"
     },
+    {
+        "name": "vernier-force-and-accel",
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/routes.json]: fixed- thanks

File: src/views/gdxfor/gdxfor.jsx [hunk 3]
```diff
@@ -0,0 +1,271 @@
+const injectIntl = require('react-intl').injectIntl;
+const intlShape = require('react-intl').intlShape;
+const FormattedMessage = require('react-intl').FormattedMessage;
+const React = require('react');
+
+
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/views/gdxfor/gdxfor.jsx]: all set

File: src/views/gdxfor/gdxfor.jsx [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+    render () {
+        return (
+            <div className="extension-landing gdxfor">
+                <ExtensionHeader
+                    renderCopy={
+                        <FlexRow className="extension-copy">
+                            <h1><img
+                                alt=""
+                                src="/images/gdxfor/gdxfor.svg"
+                            />Vernier Force & Acceleration</h1>
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/views/gdxfor/gdxfor.jsx]: Lets address in a follow-up issue

File: src/views/gdxfor/gdxfor.jsx [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+                            <FormattedMessage
+                                id="gdxfor.headerText"
+                                values={{
+                                    gdxforLink: (
+                                        <a
+                                            href="https://www.vernier.com/products/sensors/force-sensors/gdx-for/"
+                                            rel="noopener noreferrer"
+                                            target="_blank"
+                                        >
+                                            Vernier GoDirect Force & Acceleration
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/views/gdxfor/gdxfor.jsx]: Addressed in my comment below

File: src/views/gdxfor/gdxfor.jsx [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+                                                <FormattedMessage id="gdxfor.startSound" />
+                                            </strong>
+                                        )
+                                    }}
+                                />
+                            </span>
+                            <div className="step-image">
+                                <img
+                                    alt=""
+                                    src="/images/gdxfor/gdxfor-force-pushed-stack.png"
```
[the remark this answers is not in the document]
Comment (ericrosenbaum) [reply src/views/gdxfor/gdxfor.jsx]: Good point- will do!

**Comment by chrisgarrity** [comment]
@ericrosenbaum thanks for documenting the name inconsistencies.

**Pull request merged**